InnoTopicWebsite/src/app/shirt-generator/shirt-generator.page.html

```diff
@@ -0,0 +1,13 @@
+<ion-header [translucent]="true">
+  <ion-toolbar>
+    <ion-title>Shirt Generator</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content [fullscreen]="true">
+  <app-prompt-input (searchEvent)="onSearch($event)"/>
+    <app-prompt-result
+      [promptResult]="$any(result$ | async)"
+      [isLoading]="$any(isLoading$ | async)"
```

Comment:
later we have to take streaming into account ( streaming ends when observable completes )